pandunia-lekse.csv

```diff
@@ -1873,8 +1884,9 @@ macabli||||compatible (supported)||compatible|||||||||||||||
 madagasi||||Malagasy||malgache||||||||||||||malagassi|madagaskarski; malagaski
 madagasia|dex|MG||Madagascar||Madagascar||||||||||||||Madagaskar|Madagaskar
 madiaria|dex|HU||Hungary||Hungría||||||||||||||Unkari|Węgry
-mafana|||zho:麻煩 (máfan), yue:麻煩 (maa4 faan4), wuu:麻煩 (mo ve3)|bother||molestar (preocupar)||||||||||||||vaivata|przeszkodzić, przeszkadzać, robić kłopot
+mafana|||zho:麻煩 (máfan), yue:麻煩 (maa4 faan4), wuu:麻煩 (mo ve3)|bother||molestar (preocupar)||утруждать||||||||||||vaivata|przeszkodzić, przeszkadzać, robić kłopot
 mafane||||trouble (disturbance, bother)||problema (disturio)||||||||||||||vaiva (haitta)|problem, kłopot, zakłócenie
+mafano||||go through the trouble of (bother to, take the time to, inconveniently)||molestar en||утруждай|||わざわざ||||||||||
```

Comment:
The ending should be -u.

Reply:
I think it actually makes a bit more sense as an adverb. Because when you bother to do something, you still do it; the only thing "bother to" does is describe _how_ you do it: in a deliberate and inconvenient way.

Reply:
well, actually I guess the verb version does make sense semantically now I think about it. when you bother to read the instructions, you are being bothered by the act of reading the instructions. and if you say **me mafanu ze duga recete**, it make it perfectly clear that I am reading the instructions, and that fact bothers me.